app/views/classrooms/new.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="header">
+  <div class="logo">
+    <%= image_tag 'odaiba-logo_inverted_icon.png' %>
+  </div>
+</div>
+<div class="wrapper">
+  <div class="form">
+    <div class="grey-header">
+      <h3>Create a New Classroom</h3>
+    </div>
+    <%= render partial: 'shared/modal', locals: { notice: @notice } %>
+    <div class="form-text">
+      <%= simple_form_for @classroom do |f| %>
+        <%= simple_fields_for :no_model_fields do |n| %>
+          <h3>Teacher</h3>
+          <p>Please write the email of the teacher.</p>
+          <%= n.input_field :teacher_email, value: @teacher_email %>
+
+          <h3>Emails</h3>
+          <p>Please write emails of the students. Please separate by a single empty space.</p>
+          <%= n.text_area :student_emails, value: @student_emails %>
+
+          <h3>Worksheet URLs</h3>
+          <p>Paste image url of a online worksheets (separated by empty space), or click of one of the options below</p>
+          <%= n.text_area :worksheet_urls, value: @worksheet_urls %>
+          <div class="disclaimer">
+            <h3>Disclaimer:</h3>
+            <p>Please read this disclaimer before submitting the form. The odaiba app and <a href="https://www.odaiba-app.com/">https://www.odaiba-app.com/</a> are owned and operated by odaiba ("us", "we", "our").
```

Comment:
Will there be no option for email or link sharing?

Reply:
I haven't put it there yet since if I'm picturing 10 students, then it would be much easier for the teacher to send an email than to copy paste each link individually and share via another platform
what are your thoughts though?

Reply:
Good point. I know when I tried recently, it had the link copy feature, but would only copy one of the links so I had to manually copy the addresses. Guess it comes down to a matter of age again. Sending a link to students over Zoom or something is much easier than opening email inboxes for a lot of students. 

Theoretically, you could send a generic link where users see a choice of email addresses and they just click theirs and get redirected. Sure there are issues with this approach as well. 

For now probably better as it is.